src/main/java/sirius/biz/storage/layer3/ExtractArchiveJob.java

```diff
@@ -39,9 +40,29 @@
  * This uses the {@link ArchiveExtractor} so depending if 7-ZIP is enabled this supports either a whole bunch
  * of formats (rar, 7z, tar etc.) or "just" ZIP files using the Java API.
  */
-@Register
+@Register(classes = {JobFactory.class, ExtractArchiveJob.class})
```

Comment:
Hat `JobFactory` noch kein `@AutoRegister`?

Reply:
nope, once a classes list in @Register is present, this wins over all